mcc/trek/trek/trek_test.go

```diff
@@ -64,4 +65,26 @@ func TestAdd(t *testing.T) {
 	if err == nil {
 		t.Errorf("Both original and final should be present")
 	}
+	vhosts := "server {\n" +
```

Comment:
I don't think it's valuable for this PR, but we should use templates eventually. We'll need this kind of functionality again in the future so we can change it then.